test/php/library/Director/Web/Form/Element/DictionaryTest.php

```diff
@@ -32,53 +32,6 @@ public function testValidDictionary() {
         $this->assertFalse($this->dictionaryInstance->hasErrors());
     }
 
-    public function testDictionaryWithMissingKey() {
```

Comment:
shouldn't you have rebranded this test to makes sure there's not error when omitting a key??